midi/midi/decode.py

```diff
@@ -32,6 +32,13 @@ def __eq__(self, other):
                self.tone == other.tone and \
                self.octave == other.octave
 
+    def __repr__(self):
+        out_str = 'EventNote('
+        out_str += repr(self.velocity) + ', '
+        out_str += repr(self.height) + ')'
+
+        return out_str
```

Comment:
```suggestion
	return f'EventNote({self.velocity}, {self.height})'
```

Reply:
code shortened in f947ee90510cc06c7ead4eafe548daad3e73edbf